### DISCUSSION TITLE
Shortcut to launch multiple programs in a new container in the current workspace

### DISCUSSION THREAD

**Discussion by LovelyDavy** [i3/i3#5956] [Q&A]
Hi there,

I hope this isn't an obvious question, I've been trying to figure this out from the docs but I'm not sure I can figure out a way.

I'd like to be able to launch three programs (signal-desktop, whatsdesk and telegram-desktop) all at the same time in a new tabbed container in the current workspace when I hit a shortcut. Does anyone know the appropriate syntax for this? Is it even possible? 

Thank you very much for your time, I hope this is in the right place.

**Comment by orestisfl** [comment]
See layout saving https://i3wm.org/docs/layout-saving.html
